tests/auth0-session/handlers/callback.test.ts

```diff
@@ -436,60 +436,6 @@ describe('callback', () => {
     expect(header.alg).toEqual('RS256');
   });
 
-  it('should use client secret jwt on token endpoint', async () => {
```

Comment:
Apologies if this is something obvious, why are these tests being removed?

Reply:
Auth0 doesn't support client secret jwt and the edge client doesn't so I've removed support